bitnami/milvus/values.yaml

```diff
@@ -110,15 +110,24 @@ milvus:
     ## @param milvus.auth.password Milvus username password
     ##
     password: ""
-    ## @param milvus.auth.rootPassword Milvus root password
+    ## @param milvus.auth.rootOldPassword Milvus old root password, default is Milvus
     ##
-    rootPassword: ""
+    rootOldPassword: "Milvus"
+    ## @param milvus.auth.rootNewPassword Milvus new root password, used for initJob to change root password
+    ##
+    rootNewPassword: ""
```

Comment:
This change will cause a major impact, all users should change their value. What do you think about this suggestion?:

```yaml
    ## @param milvus.auth.rootPassword Milvus root password
    ##
    rootPassword: ""
    ## @param milvus.auth.defaultRootPassword Default Milvus root password, used for initJob to set root password. In case of re-installation you can set here previous password
    ##
    defaultRootPassword: "Milvus"
```

I think the code above could match your needs and it won't cause a major impact